src/app.ts

```diff
@@ -133,6 +142,16 @@ async function app() {
 
   const tank = new FSHTank(docs, config);
   await Promise.all(dependencyDefs);
+
+  // Check for StructureDefinition
```

Comment:
It looked like the task noted that we should check for ElementDefinition as well. Did you want that included here too?

Reply:
I talked about this with Chris yesterday too, and he said that he thought that checking for StructureDefinition was sufficient. I think his thinking was that if the StructureDefinition is fine, the ElementDefinition is also probably fine. If you can think of a reason that one might end up there and the other might not, I can add it back in. It isn't difficult.

Reply:
Nah, I'm good with leaving it out. Just wanted to double check that it was a conscious decision.